libs/SmartStore/src/com/salesforce/androidsdk/smartstore/phonegap/SmartStorePlugin.java

```diff
@@ -65,6 +65,7 @@
 	public static final String LIKE_KEY = "likeKey";
 	public static final String MATCH_KEY = "matchKey";
 	public static final String SMART_SQL = "smartSql";
+	public static final String ORDER_PATH = "orderPath";
```

Comment:
Previously you could only order by the field you were searching by.
But for full-text search, you can search across all indexed fields, and it didn't make sense not to have a sorting.
For backward compatibility, the javascript code uses indexPath as the orderPath when no orderPath is provided.